qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -340,6 +340,10 @@ def _follow_selected_cb(self, js_elem, tab=False):
             elem.click(click_type)
 
     def follow_selected(self, *, tab=False):
+        # Tell focustools the user is triggering this focus event.
+        self._tab.run_js_async(
+            javascript.assemble('focustools', "setUserInteracted"))
```

Comment:
Probably makes sense to add some `set_user_interacted` method to the tab API so it can be used easily from everywhere in the codebase? Probably makes sense for it to live in this caret class?